README.md

```diff
@@ -3,13 +3,13 @@
 Exercism exercises in Objective-C
 
 ## Try it!
-If you've downloaded the [command-line client](http://exercism.io/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
+If you've downloaded the [command-line client](http://exercism.org/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
 
 ```bash
 exercism fetch objective-c
 ```
 
-In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.io/account/key).
+In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.org/account/key).
```

Comment:
This link gives a 404, can you please find the equivalent replacement and use that here?